erpnext/public/js/controllers/transaction.js

```diff
@@ -1264,6 +1267,43 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 				}
 			}
 		});
+
+		const transaction_pricing_rule = this.frm.call({
```

Comment:
This should be call in the callback of apply_pricing_rule

Reply:
May i ask why?